src/encoded/commands/deploy.py

```diff
@@ -275,9 +296,10 @@ def _get_instances_tag_data(main_args):
         sys.exit(1)
     instances_tag_data['username'] = getpass.getuser()
     if instances_tag_data['name'] is None:
+        instances_tag_data['short_name'] = _short_name(instances_tag_data['branch'])
```

Comment:
Not significant: It would be nice to have Python 3.6 or higher, where we can use literal string interpolation (f-string.)